benchmarks/torchbench.py

```diff
@@ -188,6 +188,7 @@
 }
 
 DONT_CHANGE_BATCH_SIZE = {
+    "demucs",
```

Comment:
This seems to fail on CI.

Reply:
We will have to skip this one. The smallest batch size demucs support is 4, and even that is unable to fit in CI's GPU memory.